service/src/main/kotlin/fi/espoo/evaka/attendance/StaffAttendanceQueries.kt

```diff
@@ -241,3 +241,14 @@ fun Database.Read.getExternalStaffAttendancesByDateRange(unitId: DaycareId, rang
     .bind("end", range.end)
     .mapTo<ExternalAttendance>()
     .list()
+
+fun Database.Read.getGroupsForEmployee(employeeId: EmployeeId): List<GroupId> = createQuery(
+    """
+    SELECT daycare_group_id
+    FROM daycare_group_acl
+    WHERE employee_id = :employeeId
+    """.trimIndent()
+)
```

Comment:
Would be better to read group memberships for all employees in a single call, something like this:

```
private data class Foo(val employeeId: EmployeeId, val groupIds: List<GroupId>)

fun Database.Read.gepGroupsForEmployees(employeeIds: Set<EmployeeId>): Map<EmployeeId, List<GroupId>> = createQuery("""
    SELECT employee_id, json_array_agg(daycare_group_id) AS group_ids
    FROM daycare_group_acl
    WHERE employee_id = ANY(:employeeIds)
""").bind("employeeIds", employeeIds.toTypedArray()).mapTo<Foo>().associateJotainJotain()
```